Unneccessary friendly error message on min() max()

#### Most appropriate sub-area of p5.js?

- [ ] Accessibility (Web Accessibility)
- [ ] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [x] Friendly error system
- [ ] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [ ] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Details about the bug:
In the documentation for min() and max() it says "accepts any number of Number parameters" but if you give it more than 2 Number parameters there is a friendly error message. The function does work with >2 parameters and the error is unnecessary. 

Here is the error:
🌸 p5.js says: max() was expecting no more than 2 arguments, but received 6. (on line 14 in sketch.js [file:///C:/---]). (http://p5js.org/reference/#/p5/max)

- p5.js version:  p5.js v1.1.9 July 22, 2020 
- Web browser and version: Chrome 86.0.4240.183
- Operating System: Windows 10
- Steps to reproduce this:
Use min() or max() with more than 2 parameters (not in an array) 
 e.g. 
 x = max(80, 3, 5, 13, 1, 2);
 console.log(x);

I am not able to test the changes I made...
The error is coming similar to the one in this issue [#2361](https://github.com/processing/p5.js/issues/2361)

This is the error I am getting:-
```Running "eslint:build" (eslint) task
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
No parser and no filepath given, using 'babel' the parser now but this will throw an error in the future. Please specify a parser or a filepath so one can be inferred.
⏎·······` [Error/prettier/prettier]y.js:26:17: Replace `·?·'p5-test.js'` with `

1 problem
Warning: Task "eslint:build" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```

I even did a clean npm install by removing the existing node_modules folder and the npm cache

--- a/src/math/calculation.js
+++ b/src/math/calculation.js
@@ -501,7 +501,6 @@ p5.prototype.map = function(n, start1, stop1, start2, stop2, withinBounds) {
  * @return {Number}
  */
 p5.prototype.max = function(...args) {
-  p5._validateParameters('max', args);
   if (args[0] instanceof Array) {
     return Math.max.apply(null, args[0]);
   } else {
@@ -550,7 +549,6 @@ p5.prototype.max = function(...args) {
  * @return {Number}
  */
 p5.prototype.min = function(...args) {
-  p5._validateParameters('min', args);
   if (args[0] instanceof Array) {
     return Math.min.apply(null, args[0]);
   } else {

